src/components/organisms/UiModal/UiModal.vue

```diff
@@ -153,6 +156,21 @@
               </UiText>
             </slot>
           </slot>
+          <!-- @slot Use this slot to replace content template. -->
+          <slot
+            v-if="hasContent"
```

Comment:
slots shouldn't be conditioned, move this condition slot fallback

Reply:
Well, then why is there a conditioned slot on line `140`? 😅 
```
            <!-- @slot Use this slot to replace description template. -->
            <slot
              v-if="title"
```

Reply:
probably an oversight, slots should be always available

Reply:
change it  during my PR?